frontend/modules/viewerGui/viewerGui.sagas.ts

```diff
@@ -51,6 +51,19 @@ import {
 } from './viewerGui.selectors';
 
 function* fetchData({ teamspace, model }) {
+	try {
```

Comment:
use fetchSettings. Fetching model settings as the first action makes more sense than metadata keys.

Reply:
Fetching model settings is now used instead of metadata